### DISCUSSION TITLE
2021/12/28/ansible-setup-with-azure-on-a-new-mac/

### DISCUSSION THREAD

**Discussion by giscus** [russmckendrick/blog#10] [Comment]
# 2021/12/28/ansible-setup-with-azure-on-a-new-mac/

Following on from upgrading my Macs to all M1-based ones, I get around to finally needing to use Ansible; typically, I would install Python using PyEnv and then run the following …
    It should just work?!?!  pipinstall --user ansible pip install --user -r https://github.com/ansible-collections/azure/blob/dev/requirements-azure.txt ansible-galaxy collection install azure.azcollection   
… and away we go, but this time I came across a few problems.

https://www.mediaglasses.blog/2021/12/28/ansible-setup-with-azure-on-a-new-mac/

**Comment by abjutus** [comment]
great post

**Reply by abjutus** [reply]
😀

**Comment by dreits** [comment]
pip install --user -r https://github.com/ansible-collections/azure/blob/dev/requirements-azure.txt
ERROR: Invalid requirement: '<!DOCTYPE html>' (from line 8 of https://github.com/ansible-collections/azure/blob/dev/requirements-azure.txt)
